public/template.json

```diff
@@ -626,10 +628,10 @@
     },
     "culture": {
       "description": "",
-      "feature": [],
+      "features": {},
       "languages": {
-        "count": 0,
-        "options": []
+        "additional": 0,
```

Comment:
I'm not so sure about the name change. While logically it makes more sense to name it as additional, it kind of breaks consistency with how other elements in the template.json. Nothing needing to be changed just a thought.

Reply:
All cultures just grant languages, with free additional. "options" and "count" gives the impression you can pick a certain number from a list of choices. I would make an argument that all 4 values can still in principal have meaning.

```json
{
  "count": 1,
  "options": ["globin", "halfling"],
  "learn": ["common"]
  "additional": 2
}
```

This would imply you automatically learn `Common`, must learn `Goblin` or `Halfling` and you can pick any two additional languages of your choice.

Reply:
I don't agree that all of the properties have value. When it comes to `options` and `learn`, I don't agree that there is any difference at all when it comes to Common, Goblin, and Halfling. It's just that all current cultures grant Common as a convention. I think a single `defaults` or `base` property (for example) would adequately capture all of these base selections.

`count` here also seems to be completely wasted data. Is it just describing the length of the `learn` array?

I think a better approach to all of this would be a data structure like this:
```json
{
    "defaults": ["common", "goblin", "halfling"],
    "additional": 2
}
```
Or this:
```json
{
    "base": ["common", "goblin", "halfling"],
    "additional": 2
}
```

Reply:
> count here also seems to be completely wasted data. Is it just describing the length of the learn array?

It is describing the number of elements you may select from `options`. This need becomes more apparent when you look at things like class, background skill/tool proficiencies.

For, example farmer gives you `Nature`, and either `Animal Handling` or `Survival`, and can be described as such:

```json
{
    "base": ["Nature"],
    "options": ["Animal Handling", "Survival"],
    "count": 1
}```

While adapt would be (since you get to pick two skills:

```json
{
    "base": [],
    "options": ["Acrobatics", "Athletics", "History", "Insight", "Perception", "Religion", "Stealth"],
    "count": 2
}```

> I think a single defaults or base property (for example) would adequately capture all of these base selections.

I can agree to this, just let me know preferences and I'll make the change.

Reply:
> It is describing the number of elements you may select from options. This need becomes more apparent when you look at things like class, background skill/tool proficiencies.

I agree that there's value in an `options` and `count` value in those contexts, but we don't need those additional pieces of data for languages in cultures.

> I can agree to this, just let me know preferences and I'll make the change.

@NekroDarkmoon Thoughts on moving to this model for culture languages?
```json
{
    "base": ["common", "goblin", "halfling"],
    "additional": 2
}
```
I have no objections.

Reply:
I've got no objections to it either.

Reply:
Cool, looks like you can make this change then, @KageJittai 🙂